src/ctim/schemas/indicator.cljc

```diff
@@ -41,6 +41,21 @@
    (f/entry :snort_sig f/any-str))
   :description "An indicator which runs in snort...")
 
+(def-map-type Pattern
+(f/required-entries
+ (f/entry :type f/any-str)
+ (f/entry :value f/any-str)))
```

Comment:
Could you define a type with a spec and a generator that ensures the validity of the regex?
Ex: https://github.com/threatgrid/ctim/blob/master/src/ctim/schemas/common.cljc#L31

Reply:
Actually ObservablePattern specification is not needed at this time since there is no clear use case for it. Will remove it for now. It was added when the CTA integration was being discussed but that got dropped.